src/main/java/life/qbic/omero/BasicOMEROClient.java

```diff
@@ -352,7 +367,7 @@ public String getImageDownloadLink(long imageID) {
       BrowseFacility browse = gateway.getFacility(BrowseFacility.class);
       ImageData image = browse.getImage(this.securityContext, imageID);
       if (image.getFormat() != null) {
-        downloadLinkAddress = "https://" + hostname + "/omero/webgateway/archived_files/download/"
+        downloadLinkAddress = "https://" + hostname + "/webgateway/archived_files/download/"
```

Comment:
NIT: I'd suggest down the line to have dedicated linksuppliers for the individual addresses to make refactoring easier and streamlined

Reply:
I opened this issue to address this in a dedicated manner: https://github.com/qbicsoftware/omero-lib/issues/70